src/main/java/com/cloudbees/jenkins/support/filter/SensitiveContentFilter.java

```diff
@@ -26,13 +26,13 @@
 
 import hudson.Extension;
 import hudson.ExtensionList;
-import org.apache.commons.lang3.StringUtils;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 import javax.annotation.Nonnull;
 import java.util.Locale;
 import java.util.Set;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Yuck, I like using the current versions of Commons libraries. :)

Reply:
Yeah, commons-lang has a better compatibility story than snakeyaml IIRC, so it would likely be fine to add a commons-lang3 dependency, but since commons-lang 2.6 is already bundled in Jenkins core it seems easiest to just use that to avoid any issues.